charts/datadog/values.yaml

```diff
@@ -315,6 +315,13 @@ datadog:
     # datadog.systemProbe.collectDNSStats -- Enable DNS stat collection
     collectDNSStats: false
 
+    # datadog.systemProbe.maxTrackedConnections -- the maximum number of tracked connections
+    maxTrackedConnections: 65536
+
+    # datadog.systemProbe.conntrackMaxStateSize -- the maximum size of the userspace conntrack cache
+    conntrackMaxStateSize:  131072 # 2 * maxTrackedConnections by default, per  https://github.com/DataDog/datadog-agent/blob/d1c5de31e1bba72dfac459aed5ff9562c3fdcc20/pkg/process/config/config.go#L229
```

Comment:
I would suggest to add this two settings under `datadog.networkMonitoring` if they are very specific to the NPM product.